backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java

```diff
@@ -244,6 +256,34 @@ protected void createTablesLiquibaseMigration(
     }
   }
 
+  private static void resetChangeSetLogData(
+      JdbcConnection jdbcCon, String changeLogTableName, String filename) {
+    try {
+      LOGGER.info("Resetting changeset data using file {}", filename);
+      URL url = Resources.getResource(filename);
+      List<String> lines = Resources.readLines(url, StandardCharsets.UTF_8);
+      String json = String.join("", lines);
+      Gson gson = new Gson();
+      JsonReader reader = new JsonReader(new StringReader(json));
+      ChangeSetId[] changeSetIdArray = gson.fromJson(reader, ChangeSetId[].class);
+      var updateQuery = "update %s set FILENAME=? WHERE ID=?";
+      try (var statement =
+          jdbcCon.prepareStatement(String.format(updateQuery, changeLogTableName))) {
+        for (var changeSetId : changeSetIdArray) {
+          LOGGER.debug(
```

Comment:
should we have this higher than debug so that we can see it in the logs when allstate applies?

Reply:
I'm ok with moving this up to info. 👍🏼

Reply:
Not a bad idea.  Info is enabled in Allstate we should elevate to that.